### PR TITLE
Michaelwwest98 UI subsystem signoff

### PR DESCRIPTION
**Pull request by Michaelwwest98**
Hopefully this is the correct spot to create a pull request.

### CODE REVIEW

**Comment by Michaelwwest98** [comment]
Thankfully, it is the correct spot.

**Review by JesseTNRoberts — COMMENTED** [review]
A purchased website is not an appropriate method. Also, this is not nearly specific enough for this design to be buildable. 


**Buildability is a requirement for systems to be considered designed and for checkpoint 2 to be met.

**Review by JesseTNRoberts — COMMENTED** [review]
I'm unclear what this signoff is for

File: Documentation/Signoffs/User-Interface-Subsystem.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+# User Interface Subsystem
+
+## Functionality
```
Comment (JesseTNRoberts) [inline Documentation/Signoffs/User-Interface-Subsystem.md:3]: Be brief.

File: Documentation/Signoffs/User-Interface-Subsystem.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+|-------------|------------|--------|
+| Display triage results | Must display whether the victim is in the Expectant, Immediate, or Delayed category | From START Method [1] |
+| Display heartbeat data | Shall display victim’s heartbeat in beats per minute | From Conceptual Design, from Heartbeat and Respiratory Subsystem section |
+| Display respiratory data | Shall display victim’s breath rate in breaths per minute | From Conceptual Design, from Heartbeat and Respiratory Subsystem section |
+
+## Block Diagram
+![UI_Block_Diagram_1](https://github.com/Michaelwwest98/DARPA-Drone-Triage-Sensing-System/blob/cc92902a6190693c7c66fbc01ece6b466e08f3d2/Documentation/Images/UI_Block_Diagram_1.JPG)
+
+
+## Analysis
```
Comment (JesseTNRoberts) [inline Documentation/Signoffs/User-Interface-Subsystem.md:21]: I'm not reading a block of text like this. This is not analysis.

**Review by JesseTNRoberts — COMMENTED** [review]
Are these the only constraints on the system? How fast should it update?

File: Documentation/Signoffs/User-Interface-Subsystem.md (hunk 1 above)
Comment (JesseTNRoberts) [reply Documentation/Signoffs/User-Interface-Subsystem.md:3]: What is this subsystem going to do? Don't describe the surrounding systems.

**Review by JesseTNRoberts — COMMENTED** [review]
This isn't buildable and doesn't consider existing designs

**Review by JesseTNRoberts — COMMENTED** [review]
Like we talked about, I think the constraints here are too loose

**Review by JesseTNRoberts — COMMENTED** [review]
Did not address issue with constraints. How fast do the various items need to update? 

Verify that all constraints from the conceptual design are here.

How is a way to turn off related to NFPA 70? And how does NFPA 70 apply?

**Review by JesseTNRoberts — COMMENTED** [review]
How does the weight constrain you?

**Review by JesseTNRoberts — APPROVED** [review]

**Pull request merged**